ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -1174,318 +1235,9 @@ pub fn print_from_file(path: &PathBuf) -> Result<(), Box<dyn Error>> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use byteorder::WriteBytesExt;
     use num_traits::{PrimInt, ToBytes};
     use std::mem;
 
-    #[cfg(target_pointer_width = "64")]
-    fn write_native_usize(d: &mut Vec<u8>, v: usize) {
-        d.write_u64::<NativeEndian>(v as u64).unwrap(); // `as` is safe: u64 == usize
-    }
-
-    fn write_str(d: &mut Vec<u8>, s: &str) {
-        d.extend(s.as_bytes());
-        d.push(0); // null terminator.
-    }
-
-    /// Note that this test only checks valid IR encodings and not for valid IR semantics. For
-    /// example, nonsensical instruction arguments (incorrect arg counts, incorrect arg types etc.)
-    /// are not checked.
-    ///
-    /// FIXME: implement an IR validator for this purpose.
-    #[test]
-    fn deser_and_display() {
```

Comment:
I opted to delete this test because:

 - it's a paint to port
 - it always has been a pain to maintain
 - it doesn't test much that isn't already tested
 - it simplifies the operand enum, since we don't need to define `OPKIND_*` constants.